scripts/mockdata/scenarios.json

```diff
@@ -54,7 +74,7 @@
         "data": [
           { "text": "just text, no images" },
           {
-            "text": "ANOTHA ONE",
+            "text": "Conclusion: ANOTHA ONE",
```

Comment:
Perhaps I'm missing something, but in the video it shows `Conclusion: Just text no image` but here it says `Conclusion: ANOTHA ONE`. Where is the text being rendered from?

Reply:
oh lol the video was from before I did merge with vivian's latest changes

Reply:
great observation 🧐

Reply:
Ah okay, as long as the functionality is the same then I'm good with that :)